src/test/java/webdrivertasks/page/CalculatorPage.java

```diff
@@ -0,0 +1,163 @@
+        dataCenterLocationDropDown.click();
+        WebElementWaiting.waitElementFromDropDown(driver, EXPANDED_DROPDOWN_XPATH, dataCenterLocation).click();
+        return this;
+    }
+
+    public CalculatorPage selectMachineCommitedUsage(String committedUsage) {
+        WebElementWaiting.waitForVisibilityOfWebElement(driver, machineCommitedUsageDropDown);
+        machineCommitedUsageDropDown.click();
+        WebElementWaiting.waitElementFromDropDown(driver, EXPANDED_DROPDOWN_XPATH, committedUsage).click();
+        return this;
+    }
+
+    public CalculatorPage inputNumberOfNodes(String numberOfNodes) {
+        WebElementWaiting.waitForVisibilityOfWebElement(driver, numberOfNodesInputArea);
+        numberOfNodesInputArea.click();
+        numberOfNodesInputArea.sendKeys(numberOfNodes);
+        addGPUCheckbox.click();
+        return this;
+    }
+
+    public CalculatorPage selectNumberOfGPUs(String numberOfGPUs) {
+        WebElementWaiting.waitForVisibilityOfWebElement(driver, numberOfGPUDropDown);
+        numberOfGPUDropDown.click();
+        WebElementWaiting.waitElementFromDropDown(driver, EXPANDED_DROPDOWN_XPATH, numberOfGPUs).click();
+        return this;
+    }
+
+    public CalculatorPage selectLocalSSD(String localSSD) {
+        WebElementWaiting.waitForVisibilityOfWebElement(driver, localSSDDropDown);
+        localSSDDropDown.click();
+        WebElementWaiting.waitElementFromDropDown(driver, EXPANDED_DROPDOWN_XPATH, localSSD).click();
+        return this;
+    }
+
+    public CalculatorPage selectNodeCommitedUsage(String committedUsage) {
+        WebElementWaiting.waitForVisibilityOfWebElement(driver, nodeCommitedUsageDropdown);
+        nodeCommitedUsageDropdown.click();
+        WebElementWaiting.waitElementFromDropDown(driver, EXPANDED_DROPDOWN_XPATH, committedUsage).click();
+        return this;
+    }
+
+    public CalculatorPage computeEngineCost() {
+        WebElementWaiting.waitForVisibilityOfWebElement(driver, machineAddEstimationButton);
+        machineAddEstimationButton.click();
+        return this;
+    }
+
+    public CalculatorPage computeNodesCost() {
+        WebElementWaiting.waitForVisibilityOfWebElement(driver, nodeAddEstimationButton);
+        nodeAddEstimationButton.click();
+        return this;
+    }
+
+    public boolean checkTotalEstimatedCost(String estimatedCost) {
+        return WebElementWaiting.waitForPresenceOfElement(driver, FINAL_COST_LOCATOR).getText().contains(estimatedCost);
+    }
+
+    public boolean checkMachineClass(String machineClass) {
+        return WebElementWaiting.waitForVisibilityOfWebElement(driver, currentMachineClass).getText().toLowerCase().contains(machineClass.toLowerCase());
+    }
+
+    public boolean checkInstanceType(String instanceType) {
+        return WebElementWaiting.waitForVisibilityOfWebElement(driver, currentInstanceType).getText().toLowerCase().contains(instanceType.toLowerCase());
+    }
+
+}
```

Comment:
Common note: we can increase readability a bit by adding `WebElementWaiting` class to the static import. It allows not to repeat the class name